apps/client/src/app/pages/currency-spending/currency-spending/currency-spending.component.ts

```diff
@@ -86,7 +86,7 @@ export class CurrencySpendingComponent extends TeamcraftComponent {
         this.loading = true;
         return this.dataService.getItem(currency).pipe(
           map((item: ItemData) => {
-            return [].concat.apply([], item.item.tradeCurrency.map(entry => {
+            return [].concat.apply([], item.item.tradeCurrency.filter(entry => entry.npcs.length).map(entry => {
```

Comment:
```suggestion
            return [].concat.apply([], item.item.tradeCurrency.filter(entry => entry.npcs.length > 0).map(entry => {
```

It's really nitpicking, but makes maintenance easier when i'll read this in six months and will take 5 more minutes to realize it's filtering on length > 0 because I won't read this properly for the first 2 times I'll read it 😄